src/Components/SingleArticle.jsx

```diff
@@ -71,15 +91,35 @@ const SingleArticle = () => {
         <button className="vote-button">
           <ArrowBigDown className="vote-icon-down" size={30}/>
         </button>
-        <button className="action-button">
+        <button 
+          className={`action-button ${showComments ? 'active' : ''}`}
+          onClick={handleCommentsClick}
+        >
           <MessageCircle size={20} /> {article.comment_count} Comments
         </button>
         <button className="action-button">
           <Share size={20} /> Share
         </button>
       </div>
+
+      {showComments && (
```

Comment:
By factoring this out into it's own component you get the added benefit of making the returned jsx simpler to manage, ie `if(isLoading) return loading` etc